### PR TITLE
Env Tester Transform e2e tests

### PR DESCRIPTION
**Pull request by Vipinofficial11**

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java]: S3->Env Tester?

File: src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java]: s3->envtester

File: src/e2e-test/java/io/cdap/plugin/testplugins/locators/EnvTesterTransformLocators.java
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/testplugins/locators/EnvTesterTransformLocators.java]: Deduplicate -> Env tester

File: src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/package-info.java
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/package-info.java]: 2023

File: src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/TestSetupHooks.java
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/TestSetupHooks.java]: 2023

**Review by rmstar — APPROVED** [review]

File: src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java]: Updated (https://github.com/data-integrations/test-plugins/pull/1/commits/a08d1d7837fddc4a8f6f0650ab01ed97b7c96b67#diff-cfba284430cac26663d2c14aba047ec85896b229093b4d046b6f1e6267461262R23)

File: src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/testplugins/runners/TestRunner.java]: updated (https://github.com/data-integrations/test-plugins/pull/1/commits/a08d1d7837fddc4a8f6f0650ab01ed97b7c96b67#diff-cfba284430cac26663d2c14aba047ec85896b229093b4d046b6f1e6267461262R31)

File: src/e2e-test/java/io/cdap/plugin/testplugins/locators/EnvTesterTransformLocators.java
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/testplugins/locators/EnvTesterTransformLocators.java]: Updated (https://github.com/data-integrations/test-plugins/pull/1/commits/a08d1d7837fddc4a8f6f0650ab01ed97b7c96b67#diff-bdb6f29e40cfa117d8df43d2ebaadbab940329f7421cea94334e6359f5c81eb1R24)

File: src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/TestSetupHooks.java
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/TestSetupHooks.java]: updated (a08d1d7837fddc4a8f6f0650ab01ed97b7c96b67)

File: src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/package-info.java
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/testplugins/common/stepsdesign/package-info.java]: updated (a08d1d7837fddc4a8f6f0650ab01ed97b7c96b67)

**Comment by Vipinofficial11** [comment]
@rmstar Please don't merge the PR, ill merge it once we are able to run few successful test builds on HDF.

**Comment by rmstar** [comment]
> @rmstar Please don't merge the PR, ill merge it once we are able to run few successful test builds on HDF.

Ok, sg. Please squash commits when you are ready to merge

**Comment by Vipinofficial11** [comment]
Validated the plugin upload script and e2e tests on Cloud Build.

Reports: 

Screenshot 2024-01-05 at 16 50 15

**Comment by Vipinofficial11** [comment]
Tests passed in 6.10.0 regression as well, changes look good merging the PR.

**Pull request merged**
